src/codeql.ts

```diff
@@ -370,3 +352,36 @@ async function getRemoteQueryPackDefaultQuery(
 
   return queries[0];
 }
+
+/**
+ * Checks that `codeql database run-queries` produced exactly one BQRS result file.
```

Comment:
How about this for a description: `Finds the BQRS file fora database and ensures that exactly one is produced` or something like that?

Reply:
Thanks, done! 👍🏽